src/frontend/apps/impress/src/features/docs/doc-table-content/components/Heading.tsx

```diff
@@ -46,6 +46,7 @@ export const Heading = ({
           block: 'start',
         });
       }}
+      $css="text-align: left;"
```

Comment:
I did not understand the problem from the commit message. Can you try explaining more and going into what is the problem. Why not targeting the header element is an issue?

Reply:
I improve the commit, plus add screenshots of the problems solved.

Reply:
thanks :star_struck: I understood everything now!